Automattic/jetpack
# My Jetpack: made backup card stats readable by screen readers

Fixes https://github.com/Automattic/jetpack/issues/35481

## Proposed changes:

On the My Jetpack page, the stats of the VaultPress Backup card are not read properly by screen readers. This PR fixes it.

_VoiceOver reads the numbers without context_
<img width="295" alt="Screenshot 2024-02-20 at 10 54 55 AM" src="https://github.com/Automattic/jetpack/assets/1620183/af392cf5-8781-4a89-847b-5b31ab18a440">

_Note: the only other card containing stats is the VideoPress one which, by looking at the implementation, seems accessible by screen readers._

### Other information:


## Jetpack product discussion


Project thread: pf5801-xS-p2

## Does this pull request change what data or activity we track or use?


No.

## Testing instructions:






- Spin up a test site with this branch
- If you run the branch locally, make sure to build `packages/my-jetpack`
- Visit `/wp-admin/admin.php?page=my-jetpack`
- Notice the VaultPress Backup card hasn't changed visually
- If you can use a screen reader, ensure that it reads the name of the stat along with its value

https://github.com/Automattic/jetpack/assets/1620183/1777315c-5008-4d61-8a59-5864494cf8eb

- Resize the window to make the "more" indicator appear
<img width="285" alt="Screenshot 2024-02-20 at 10 54 51 AM" src="https://github.com/Automattic/jetpack/assets/1620183/9fa1c020-b879-4718-8e8a-c3c84c7f47c4">

- Repeat the same steps
- The screen reader should read all stats, even the ones that are visually hidden
